Dockerfile

```diff
@@ -1,11 +1,14 @@
-FROM docker.io/fukamachi/qlot
+FROM alpine:latest
+
+WORKDIR /app
 
 COPY . .
 
-RUN apt-get update && apt-get install gcc libncurses-dev -y
+RUN apk add --no-cache curl bash build-base ncurses-dev sbcl git
 
-RUN qlot install
+RUN curl -L https://qlot.tech/installer | bash
 
-RUN qlot exec sbcl --noinform --load scripts/build-ncurses.lisp
+RUN qlot install && \
+    qlot exec sbcl --noinform --load scripts/build-ncurses.lisp
```

Comment:
sbcl installed from apt is out of date, so we need to use fukamachi/qlot or fukamachi/sbcl

Reply:
also, thank you very much for the excellent PR.

Reply:
Alpine, it only looks out by 1 minor version https://pkgs.alpinelinux.org/package/edge/community/x86_64/sbcl :)

Reply:
I didn't know that.
That's good then.